projects/1.16/assets/snail-mail/snailmail/lang/zh_cn.json

```
@@ -0,0 +1,46 @@
+{
+    "itemGroup.snailmail.main": "蜗牛信箱",
+    "entity.snailmail.snail_man": "牛递员",
+    "container.snailmail.snail_box": "蜗牛信箱",
+    "container.snailmail.envelope": "信封",
+    "container.snailmail.envelope.textfield.to": "收信地",
+    "container.snailmail.envelope.textfield.from": "发信地",
+    "block.snailmail.snail_box": "蜗牛信箱",
+    "item.snailmail.envelope_open": "信封 (已拆)",
+    "item.snailmail.envelope_closed": "信封 (未拆)",
+    "item.snailmail.stamp": "邮票",
+    "tooltip.snailmail.to": "送往: %s",
+    "tooltip.snailmail.from": "来自: %s",
+    "tooltip.snailmail.stamped": "已盖邮戳",
+    "tooltip.snailmail.delivery_failed": "递送失败",
+    "modal.snailmail.no_envelope": "服务器上的插槽中没有信封...可能发生了错误?",

```

```suggestion
    "modal.snailmail.no_envelope": "服务器上的邮件槽中没有信封……可能发生了错误？",
```
请使用`……`(中文输入法下shift+6)表示省略号（

---

已修改，感谢指教